doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -182,6 +182,12 @@ Bluetooth libraries and services
 
   * Fixed discovery of empty services.
 
+* :ref:`ble_rpc` library:
+
+  * Added missing host callback handlers for the CCC descriptor write and match operation.
+  * Fixed the attribute write callback serialization.
+  * Fixed the GATT dynamic service unregister function call serialization.
```

Comment:
These are all a bit hard to read: Which is an attribute to what. Having long list of substantives makes it difficult to understand.
Could these be for example something like this?

* Added (do we need to say missing, if we added something) host callback handlers for the ``write`` and ``match`` operations of the CCC descriptor.
* Fixed the serialization of the ``write`` callback attribute.
* Fixed the serialization of ... (cannot really say what defines what)

I may have misunderstood all of this totally.

Reply:
I rewrited it a little. I hope it is understandable now